include/klee/Interpreter.h

```diff
@@ -14,6 +14,9 @@
 #include <set>
 #include <string>
 #include <vector>
```

Comment:
I wonder if this will work if you `make install`. The lib/Core/ files won't be copied in the installation folder. Maybe you could just declare classes as:
```
class TimingSolver;
class StatsTracker;
class SpecialFunctionHandler;
```

Reply:
'make install' seems to work. With the forward declarations, there will be errors in main.cpp when one of these classes is used, e.g. https://github.com/leonbett/klee/blob/56889ff49281fc21d826246fca244487be420232/tools/klee/main.cpp#L1203

Reply:
good catch. I did not see it was actually used. I thought it was only passed as a pointer to the constructor.